owncloudApp/src/main/res/layout/preview_audio_fragment.xml

```diff
@@ -29,32 +29,51 @@
 
     <FrameLayout
         android:id="@+id/visual_area"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="0dp"
-        android:layout_alignParentTop="true"
         android:layout_above="@+id/media_controller"
-        >
+        android:layout_alignParentTop="true"
```

Comment:
`layout_above` and `layout_alignParentTop` are not needed anymore, are they? We moved to ConstraintLayout 🍻